_data/simple-icons.json

```
@@ -1110,6 +1110,12 @@
             "source": "https://www.arm.com",
             "guidelines": "https://www.arm.com/company/policies/trademarks/guidelines-corporate-logo"
         },
+        {
+            "title": "Ars Technica",
+            "hex": "FF4E00",
+            "source": "https://arstechnica.com",
+            "guidelines": "https://brandingguide.app/brand/ars-technica"
+        },
```

```suggestion
        {
            "title": "Ars Technica",
            "hex": "FF4E00",
            "source": "https://arstechnica.com"
        },
```

I would remove the 'guidelines' link - as I can't see any official affiliation with Ars Technica.

---

I remember as it being a redirect from their official website. I cannot find it anymore so I'll remove the guidelines part.